docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -268,7 +270,16 @@ Model Scope
 The scope of the Vivarium modeling strategy detailed in this document is to 
 sample the hemoglobin concentration for an individual simulant (who is not a 
 woman of reproductive age) and evaluate if that simulant's hemoglobin 
-concentration will respond to iron supplementation.
+concentration will respond to iron supplementation (iron responsive). 
+
+The modeling strategy detailed in this document aims to evaluate *all* iron 
```

Comment:
Maybe 'the modeling strategy detailed **in this section**', to distinguish from the GBD strategy described elsewhere

Reply:
Good point! I will add